### ISSUE TITLE
ValueError while running  "allesfitter.show_initial_guess('allesfit')"

### ISSUE THREAD

**Issue by rajph0** [MNGuenther/allesfitter#62]
I was trying to run  this "allesfitter.show_initial_guess('allesfit')"  to get best-fit of a flare but  I am getting ValueError while running this code although in my  "TESS.csv" has no Nan values :  
I am using python==3.8.0 .
 '/home/rajib/allesfitter-master/mho4/allesfit' is the directory of ''TESS.csv' file.

code:
allesfitter.show_initial_guess('/home/rajib/allesfitter-master/mho4/allesfit')
allesfitter.ns_fit('/home/rajib/allesfitter-master/mho4/allesfit')
allesfitter.ns_output('/home/rajib/allesfitter-master/mho4/allesfit')

Filling the Basement

allesfitter version
---------------------
v1.2.10
OrderedDict([('user-given:', ''),
             ('companions_phot', ['b']),
             ('companions_rv', []),
             ('inst_phot', ['TESS']),
             ('inst_rv', []),
             ('multiprocess', True),
             ('multiprocess_cores', 'all'),
             ('fast_fit', 'False'),
             ('fast_fit_width', ''),
             ('secondary_eclipse', 'True'),
             ('phase_curve', 'True'),
             ('phase_curve_style', 'GP'),
             ('mcmc_nwalkers', '100'),
             ('mcmc_total_steps', '2000'),
             ('mcmc_burn_steps', '1000'),
             ('mcmc_thin_by', '1'),
             ('ns_modus', 'dynamic'),
             ('ns_nlive', '500'),
             ('ns_bound', 'single'),
             ('ns_sample', 'rwalk'),
             ('ns_tol', '0.01'),
             ('host_ld_law_TESS', 'quad'),
             ('baseline_flux_TESS', 'hybrid_offset'),
             ('error_flux_TESS', 'sample'),
             ('t_exp_TESS', ''),
             ('t_exp_n_int_TESS', ''),
             ('host_N_spots_TESS', ''),
             ('N_flares', '1'),
             ('fit_ttvs', 'False'),
             ('host_grid_TESS', 'default'),
             ('host_shape_TESS', 'sphere'),
             ('automatically set:', ''),
             ('time_format', 'BJD_TDB'),
             ('inst_rv2', []),
             ('companions_all', ['b']),
             ('inst_all', ['TESS']),
             ('print_progress', True),
             ('shift_epoch', True),
             ('inst_for_b_epoch', ['TESS'])])
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
Cell In [16], line 3
      1 #one flare
----> 3 allesfitter.show_initial_guess('/home/rajib/allesfitter-master/mho4/allesfit')
      4 allesfitter.ns_fit('/home/rajib/allesfitter-master/mho4/allesfit')
      5 allesfitter.ns_output('/home/rajib/allesfitter-master/mho4/allesfit')

File ~/.local/lib/python3.8/site-packages/allesfitter/general_output.py:1025, in show_initial_guess(datadir, quiet, do_logprint, do_plot, return_figs, kwargs_dict)
   1023 def show_initial_guess(datadir, quiet=False, do_logprint=True, do_plot=True, return_figs=False, kwargs_dict=None):
   1024     #::: init
-> 1025     config.init(datadir, quiet=quiet)    
   1027     #::: show initial guess
   1028     if do_logprint: 

File ~/.local/lib/python3.8/site-packages/allesfitter/config.py:27, in init(datadir, quiet)
     25 def init(datadir, quiet=False):
     26     global BASEMENT
---> 27     BASEMENT = Basement(datadir, quiet=quiet)

File ~/.local/lib/python3.8/site-packages/allesfitter/basement.py:93, in Basement.__init__(self, datadir, quiet)
     91 self.load_settings()
     92 self.load_params()
---> 93 self.load_data()
     95 if self.settings['shift_epoch']:
     96     try:

File ~/.local/lib/python3.8/site-packages/allesfitter/basement.py:981, in Basement.load_data(self)
    979     custom_series = np.zeros_like(time)
    980 if any(np.isnan(time*flux*flux_err*custom_series)):
--> 981     raise ValueError('There are NaN values in "'+inst+'.csv". Please make sure everything is fine with your data, then exclude these rows from the file and restart.')
    982 if any(flux_err==0):
    983     raise ValueError('There are uncertainties with values of 0 in "'+inst+'.csv". Please make sure everything is fine with your data, then exclude these rows from the file and restart.')

ValueError: There are NaN values in "TESS.csv". Please make sure everything is fine with your data, then exclude these rows from the file and restart.

**Comment by MNGuenther** [comment]
Hey @rajph0 thanks for getting in touch. Without seeing your TESS.csv file, I can only guess. But the most common issue is that you might have a pandas-style header row (e.g., `time,flux,flux_err`) rather than a numpy-style header row (e.g. `#time,flux,flux_err`) in your file. Please let me know if that solves it, and if not, please attach your TESS.csv file here or email it to me directly, and I will check it for you!

**Comment by MNGuenther** [comment]
Hey @rajph0, I assume this is linked to your params.csv and settings.csv files. How did you set these up?

There seem to be a lot of rows in both files that should not be there for a pure flare fitting case. For example, your files seem to include all kinds of planetary parameters, limb darkening, phase curve settings, etc. that are not applicable to your case.

Have a look at the streamlined versions of the params.csv and settings.csv files [here](https://github.com/MNGuenther/allesfitter/tree/master/tutorials/08_stellar_flares).

Basically, once you delete all the rows you don't need, it should run.

**Comment by rajph0** [comment]
Hi @MNGuenther ,

Thanks for quick responses. Yeah, after deleted the unnecessary rows, it work fine.
